### ISSUE TITLE
opencv4.0.1 load freeze model error

### ISSUE THREAD

**Issue by opentld** [argman/EAST#268]
Hi friends:
I download official "Models trained on ICDAR 2013 (training set) + ICDAR 2015 (training set): BaiduYun http://pan.baidu.com/s/1jHWDrYQ " model and try to freeze it to pb model.
I found the output layers must be "model_0/feature_fusion/Conv_7/Sigmoid" and "model_0/feature_fusion/concat_3", and successfully freeze it to pb model.
But, when I use opencv4.0.1 dnn to read the net,error occurs:
OpenCV: terminate handler is called! The last OpenCV error is:
OpenCV(4.0.1) Error: Unspecified error (Input layer not found: resnet_v1_50/conv1/BatchNorm/beta) in cv::dnn::dnn4_v20181221::`anonymous-namespace'::TFImporter::connect, file e:\sourcecodes\opencv\opencv-4.0.1\modules\dnn\src\tensorflow\tf_importer.cpp, line 497

Why this happens? Why openCV can not find the input layer "resnet_v1_50/conv1/BatchNorm/beta" ?

many thanks for your help !

**Comment by SpringRainLu** [comment]
i have the same prolem , do you have a solutoin@opentld

**Comment by XiaoLaoDi** [comment]
@SpringRainLu @opentld same problem, have you find the solution ?

**Comment by XiaoLaoDi** [comment]
@argman hey, nice work, I am using the following code to convert ckpt to a pb model. But cannot read the pb model from cv2.dnn.readNet() api, do you have any suggestions ? 

`import tensorflow as tf
from tensorflow.python.framework import graph_util


def ckpt2pb():
    with tf.Graph().as_default() as graph_old:
        isess = tf.InteractiveSession()

        ckpt_filename = './tmp/model.ckpt-1011'
        isess.run(tf.global_variables_initializer())
        saver = tf.train.import_meta_graph(ckpt_filename+'.meta', clear_devices=True)
        saver.restore(isess, ckpt_filename)

        constant_graph = graph_util.convert_variables_to_constants(isess, isess.graph_def, ["model_0/feature_fusion/Conv_7/Sigmoid", "model_0/feature_fusion/concat_3"] , variable_names_blacklist="global_step")
        constant_graph = graph_util.remove_training_nodes(constant_graph)

        with tf.gfile.GFile('./tmp/model.pb', mode='wb') as f:
            f.write(constant_graph.SerializeToString())

ckpt2pb()`
